src/__tests__/utils/location-helper.spec.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import React from "react";
+import { OneMapService } from "../../services";
+import { LocationHelper } from "../../components/fields/location-field/location-helper";
+
+describe("when reverseGeoCode returns an error", () => {
+	let mockError;
+	const mockOnError = jest.fn();
+	const mockRef: React.MutableRefObject<AbortController | null> = {
+		current: null,
+	};
+
+	beforeEach(() => {
+		jest.resetAllMocks();
```

Comment:
as your assertions are nested within a try... catch block, it may not run if your function resolves successfully.
you can add `expect.hasAssertions();` to make sure each test has some assertions.

Reply:
Thanks, I've added this in each of the test cases